debian/builds

```diff
@@ -0,0 +1,8 @@
+#! /bin/bash
```

Comment:
Our code conventions says to use bash.

https://github.com/kubernetes/kubernetes/blob/master/docs/devel/coding-conventions.md#code-conventions

https://google.github.io/styleguide/shell.xml?showone=Which_Shell_to_Use#Which_Shell_to_Use